### ISSUE TITLE
Cropper doesn't take parent's height when parent is relative

### ISSUE THREAD

**Issue by Daedren** [Norserium/vue-advanced-cropper#169]
Hello! I'm coming across an issue when trying to use the cropper.

When a cropper with `height: 100%;` is inside a parent with an proper fixed height, it'll contract to the parent's height, this is as it should and is fine.

However, if the parent's height is relative, or is a flex item, the cropper will ignore the space it is allocated.

https://codesandbox.io/s/vue-advanced-cropper-basic-jfy5w?file=/src/App.vue&resolutionWidth=320&resolutionHeight=300

In this example, if you remove the `<cropper>` from the template, you'll see the parent has the remaining flex size since it has `flex: 1`, but the cropper was ignoring that, despite having `height: 100%`.

**Comment by Norserium** [comment]
@Daedren, hello! Are you sure, that you provide the correct link to the codesandbox?

**Comment by Daedren** [comment]
Oh whoops, I messed up.

I've updated the original comment with the correct link.

**Comment by Norserium** [comment]
@Daedren, do you want to get this result?
![image](https://user-images.githubusercontent.com/9862719/147822395-96f253ec-baf1-4aad-bfd6-bfa156c01196.png)

**Comment by Daedren** [comment]
Yeah that would be my objective. Am I messing something up on my end?

**Comment by Norserium** [comment]
@Daedren, well, there is the detail, that you actually missed.

First of all, the cropper tries to stretch the container as much as possible to fit the image completely by setting the fixed height and width:
![image](https://user-images.githubusercontent.com/9862719/147822744-f91208c2-b65d-4739-aaad-85f8fb9ce0df.png)

As a result, the parent elements (`.cropper` and then `#cropperParent`) becomes the same size as the stretcher.

It's not what you expected. Is it? 

But it's the right behavior according to the standards, because the default value of `min-height` is `auto`. It means that flex item cannot be smaller than the size of its content along the main axis. You can read about it [here](https://medium.com/the-crazy-coder/the-mystery-of-css-flex-layout-items-shrinking-8748145e96d9).

Therefore, just set `min-height: 0` for `#cropperParent`. In this case stretcher won't be able to make the cropper more than the free space, and that's what you want to get. There is [the example](https://codesandbox.io/s/vue-advanced-cropper-issue-169-fixed-0mxii).

Also, it's not recommended to use identifiers to styling elements. They are not intended for it.

**Comment by Norserium** [comment]
@Daedren, any news?

**Comment by Daedren** [comment]
Apologies, but yes, that solved it.
Thank you for the quick replies, I've learned quite a few things today.

**Comment by Norserium** [comment]
@Daedren, it's all right. You are welcome!
